packages/react/src/components/dropdown/Dropdown.module.css

```diff
@@ -0,0 +1,333 @@
+@import "hds-core";
+
+.root {
+  --border-radius: 2px;
+  --border-width: 2px;
+  --outline-width: 3px;
+  --dropdown-height: 52px;
```

Comment:
Pixels for borders and outline is a design decision (we use relative units for everything), but should the dropdown-height be relative instead of pixels? That way it would scale with the font-size.